cranelift/codegen/src/nan_canonicalization.rs

```diff
@@ -38,6 +38,8 @@ fn is_fp_arith(pos: &mut FuncCursor, inst: Inst) -> bool {
                 || opcode == Opcode::Nearest
                 || opcode == Opcode::Sqrt
                 || opcode == Opcode::Trunc
+                || opcode == Opcode::Fdemote
+                || opcode == Opcode::Fpromote
```

Comment:
Should [`fvpromote_low`](https://docs.rs/cranelift-codegen/latest/cranelift_codegen/ir/trait.InstBuilder.html#method.fvpromote_low) also be included here?

Reply:
Good catch! That did indeed previously give different answers for x86 and riscv64 for the test checked in now.